File: src/topcoder/Template.java
package topcoder;

import java.util.*;
import java.io.*;

public class Template {

    static PrintWriter pw;

    void solve() {

    }

    public void run() {

    }

    void pr(int num) {
        pw.println(num);
    }

    void pr(long num) {
        pw.println(num);
    }

    void pr(double num) {
        pw.println(num);
    }

    void pr(String s) {
        pw.println(s);
    }

    void pr(char c) {
        pw.println(c);
    }

    public static void main(String[] args) {
        pw = new PrintWriter(System.out);
        new Template().run();
        pw.close();
    }

    void tr(Object... o) {
        pw.println(Arrays.deepToString(o));
    }
}

File: src/codeforce/kotlin_heros/template.kt
package codeforce.kotlin_heros

import java.util.*
import java.io.*

fun solve(n: Int, a: List<Int>) {

}

val pw = PrintWriter(System.out)
fun main() {
    var t = ni()
    repeat(t) {
        var n = ni()
        var a = nai()
        solve(n, a)
    }
    pw.close()
}

private fun line() = readLine()!!
private fun ni() = line().toInt()
private fun nl() = line().toLong()
private fun nd() = line().toDouble()
private fun nas() = line().split(" ")
private fun nai() = nas().map { it.toInt() }
private fun nal() = nas().map { it.toLong() }
private fun nad() = nas().map { it.toDouble() }
private fun prt(e: Any) = pw.print(e)
private fun pr(e: Any) = pw.println(e)
private fun tr(vararg x: Any) = System.out.println(Arrays.deepToString(x))